### PR TITLE
[ADD] Dockerfile and Docker installations in README

### PR DESCRIPTION
**Pull request by arkwrn**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Pulling base image
+FROM ubuntu:18.04
+MAINTAINER Arie Kurniawan "contact.arkwrn@gmail.com"
```
Comment (0xInfection) [inline docker/Dockerfile]: I prefer not including any credits in-file. As a matter of recognition I'll give you credits on AUTHORS file.

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      default-libmysqlclient-dev \
+      python-xmpp \
+    && \
+    rm -rf /var/lib/apt/lists/* && \
+    apt-get clean
+
+# Installing TIDoS-Framework
+RUN git clone https://github.com/0xinfection/tidos-framework.git && \
+    cd tidos-framework && \
+    if [ $(grep 'dnslib' requirements.txt | wc -l) -eq 0 ]; then $(echo 'dnslib' >> requirements.txt); fi && \
```
Comment (0xInfection) [inline docker/Dockerfile]: DNSLib is not really a requirement as far as I see.

File: README.md [hunk 3]
```diff
@@ -89,6 +89,19 @@ Thats it, You now have TIDoS at your service. Fire it up using:
 python2 tidos.py
 ```
 
+#### Docker image :
+
+You can pull dockerized TIDoS-Framework from dockerhub :
```
Comment (0xInfection) [inline README.md]: References to dockerfile from the original repo is required, not the external one which isn't controlled by project author.

**Comment by 0xInfection** [comment]
Closing due to inactivity. A proper docker file will be provided in v2.

File: docker/Dockerfile (hunk 1 above)
Comment (arkwrn) [reply docker/Dockerfile]: Agree with You, sorry i'll delete it

File: README.md (hunk 3 above)
Comment (arkwrn) [reply README.md]: Noted. switch to manual deployment instead of pulling from official repo

**Pull request closed without merging**